cmd/new.go

```diff
@@ -367,6 +348,76 @@ func errorIfNotEmptyDirectory(path string) error {
 	return nil
 }
 
+// getStack gets a stack or returns nil if the stack doesn't exist.
+// name and description are set if not empty and those values are available from the existing stack.
+func getStack(stack string, name *string, description *string, opts display.Options) (backend.Stack, error) {
```

Comment:
Having two ref parameters as return values feels a little weird, can this not just return the stack as well as the name and description and possibly an error?

Reply:
Done